src/use/scroll-state.ts

```diff
@@ -0,0 +1,49 @@
+import { computed, onMounted, Ref, ref, onUnmounted } from '@vue/composition-api';
+import Vue from 'vue';
+
+export function useScrollState(target: Ref<HTMLElement | null>) {
```

Comment:
Why do we need `target` to be able to be `null` here? As far as I can tell, there are no type errors if it can't be, and on the following methods we are using `target.value` without null check

Reply:
True. Will be fixed in the following PR :)